### DISCUSSION TITLE
Explanation of the columns in plotman status

### DISCUSSION THREAD

**Discussion by chadmccune** [ericaltendorf/plotman#284] [Q&A]
Specifically, what is wall, user, sys, and io?

**Comment by thomann061** [comment]
**wall**: the elapsed time the job ran for.
**user**: time spent by normal processes executing in user mode; on Linux this also includes guest time
**sys**tem: time spent by processes executing in kernel mode
**io**wait: time spent waiting for I/O to complete. This is not accounted in idle time counter.

**Reply by gaukas** [reply]
Thanks. Your explanation really helps a lot.
